scripts/requirements-darwin.txt

```diff
@@ -0,0 +1,9 @@
+python=3.7.7;hash=b703318dd9eb7bc5e8d97c8b95e9445f754ca22f
```

Comment:
Rename the file to `tool-versions-darwin.txt`

Reply:
done